src/app/services/request-exporter.service.ts

```diff
@@ -76,23 +75,14 @@ export class RequestExportInstance {
             }
             this.curProgress++;
         }
-        const fileName = `Token-ATM-Requests-Export-${
-            this.fileName ? sanitizeFileName(this.fileName) + '-' : '' + format(new Date(), 'MM-dd-yyyy-HH-mm-ss')
-        }`;
+        const fixes = { prefix: 'Token-ATM-Requests-Export', suffix: '' };
+        const fName = this.fileName ? this.fileName : '';
```

Comment:
[The nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) could be used here.